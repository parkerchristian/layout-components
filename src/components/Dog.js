import React from 'react';

export default function Dog() {
  return (
    <dl>
      <dt>Name</dt>
      <dd>Jimmy</dd>

      <dt>Age</dt>
      <dd>1</dd>

      <dt>Weight</dt>
      <dd>100 lbs</dd>
    </dl>
  );
}
