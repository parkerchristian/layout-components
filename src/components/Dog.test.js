import React from 'react';
import { shallow } from 'enzyme';
import Dog from './Dog';

describe('Dog component', () => {
  it('renders a Dog', () => {
    const wrapper = shallow(<Dog />);
    expect(wrapper).toMatchSnapshot();
  });
});
