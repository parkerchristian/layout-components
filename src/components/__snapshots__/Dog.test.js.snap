// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog component renders a Dog 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Jimmy
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    1
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    100 lbs
  </dd>
</dl>
`;
